Find_The_Second_Largest_Number.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Create a function that takes a list of numbers and returns the second largest number.
+
+Examples
+
+second_largest([10, 40, 30, 20, 50]) ➞ 40
+
+second_largest([25, 143, 89, 13, 105]) ➞ 105
+
+second_largest([54, 23, 11, 17, 10]) ➞ 23
+=end
+
+
+def second_largest(array)

```

Comment:
why do we need to sort it to find that out? 

you can come up with an algorithm where you don't have to sort it. you can just loop through the array once and find the second largest number. 

Think about how you would find the largest number in an array without sorting. You need to evolve that code to find the second largest number.